runhouse/main.py

```diff
@@ -679,6 +350,182 @@ def cluster_list(
         print_sky_clusters_msg(len(sky_clusters))
 
 
+@cluster_app.command("keep-warm")
+def cluster_keep_warm(
+    cluster_name: str,
+    mins: Optional[int] = typer.Option(
+        -1, help="Amount of time (in min) to keep the cluster warm after inactivity."
+    ),
+):
+    """Keep the cluster warm for given number of minutes after inactivity.
+
+    Example:
+        ``$ runhouse cluster keep-warm /sashab/rh-basic-cpu``
+
+    """
+
+    try:
+        current_cluster = rh.cluster(name=cluster_name)
+        if not current_cluster.is_up():
+            print_bring_cluster_up_msg(cluster_name=cluster_name)
+            return
+        current_cluster.keep_warm(mins=mins)
+        # TODO: keep warm when we are in the cluster?
```

Comment:
i think we should support running this on a cluster, should be easy if we just detect that at the beginning of the command